src/DotNetCore.CAP.AzureServiceBus/AzureServiceBusConsumerClient.cs

```diff
@@ -8,16 +8,17 @@
 using System.Threading.Tasks;
 using Azure.Messaging.ServiceBus;
 using Azure.Messaging.ServiceBus.Administration;
+using DotNetCore.CAP.AzureServiceBus.Consumer;
 using DotNetCore.CAP.Messages;
 using DotNetCore.CAP.Transport;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
 
 namespace DotNetCore.CAP.AzureServiceBus;
 
-internal sealed class AzureServiceBusConsumerClient : IConsumerClient
+internal class AzureServiceBusConsumerClient : IConsumerClient
 {
-    private readonly AzureServiceBusOptions _asbOptions;
+    private AzureServiceBusOptions _asbOptions;
```

Comment:
Should we mutate this field here or create a new one with the CustomConsumer configurations instead? It's main purpose is to reflect injected values.